packages/common/src/eips.ts

```diff
@@ -542,4 +542,13 @@ export const EIPs: EIPsDict = {
       },
     },
   },
+  7685: {
+    comment: 'General purpose execution layer requests',
+    url: 'https://eips.ethereum.org/EIPS/eip-7685',
+    status: Status.Draft,
+    // TODO: Set correct minimum hardfork
```

Comment:
TODO left here, I would assume that this can be done from Merge.

Reply:
This one isn't obvious to me.  Unless we do some more structured work around `raw()` function in the `Blockheader` class, not sure we can have less than Cancun here.  See https://github.com/ethereumjs/ethereumjs-monorepo/pull/3372#discussion_r1581698444

Reply:
Hmm right this is a good point, I think for `raw` it will work: 

```typescript
    if (this.common.isActivatedEIP(4844)) {
      rawItems.push(bigIntToUnpaddedBytes(this.blobGasUsed!))
      rawItems.push(bigIntToUnpaddedBytes(this.excessBlobGas!))
    }
    if (this.common.isActivatedEIP(4788)) {
      rawItems.push(this.parentBeaconBlockRoot!)
    }
    if (this.common.isActivatedEIP(7685) === true) {
      rawItems.push(this.requestsRoot!)
    }
```

However we get in trouble in `valuesArrayToHeaderData` since this assumes that 4788, 4844 is active in case we activate 7685. So you are correct, in our current codebase the minimum is Cancun.

Reply:
It doesn't really even make sense in `raw` since we aren't checking to verify that the items we're pushing into the array are in the correct position.  The blob gas fields should always come before the `parentBeaconBlockRoot` but if you have 4788 activated and not 4844, you would get the parentBeaconBlockRoot in the position where our code expects the blob related fields to be